pkg/common/utils/utils.go

```diff
@@ -102,6 +105,22 @@ func GetApplicationIDFromPod(pod *v1.Pod) (string, error) {
 		pod.Spec.String())
 }
 
+func GetNamespaceQuotaFromAnnotation(namespaceObj *v1.Namespace) *si.Resource {
+	log.Logger.Info("### GetNamespaceQuotaFromAnnotation")
+	// retrieve resource quota info from annotations
+	cpuQuota := namespaceObj.Annotations["yunikorn.apache.org/queue.max.cpu"]
+	memQuota := namespaceObj.Annotations["yunikorn.apache.org/queue.max.memory"]
```

Comment:
These should be referring to namespace not to queue. The fact that we map that in the core to a queue is beside the point. On the k8s side it is applying to a namespace.

Reply:
Makes sense.